Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -123,6 +119,9 @@ - (void)start
             }
 
             _toxav_iterate(strongSelf.toxAV);
+
+            uint64_t nextIterate = _toxav_iteration_interval(strongSelf.toxAV) * (NSEC_PER_SEC / 1000);
+            dispatch_source_set_timer(strongSelf.timer, dispatch_walltime(NULL, nextIterate), DISPATCH_TIME_FOREVER, nextIterate / 5);
```

Comment:
It think it would be better to save update timer only if nextIterate isn't equal to previous one.

```objc
uint64_t nextIterate = _toxav_iteration_interval(strongSelf.toxAV) * (NSEC_PER_SEC / 1000);
if (nextIterate != previousIterate) {
    previousIterate = nextIterate;
    dispatch_source_set_timer(strongSelf.timer, dispatch_walltime(NULL, nextIterate), DISPATCH_TIME_FOREVER, nextIterate / 5);
}
```

Also you can move `dispatch_source_set_timer` to separate method or create a block with it to avoid code duplication.

```objc
updateTimerInterval(nextIterate);
```